### ISSUE TITLE
100% cpu usage

### ISSUE THREAD

**Issue by DaddySpin** [P3NG00/NetheriteHorseArmorFabric#3]
The newest update to the mod makes my server always use 100% cpu and when i stop my server it doesnt stop and i have to kill the server process.

Without Mod:
![chrome_2020-08-05_00-33-30](https://user-images.githubusercontent.com/34575655/89375765-fb30f080-d6b3-11ea-9b73-b084c141df4b.png)

With Mod:
![chrome_2020-08-05_00-37-42](https://user-images.githubusercontent.com/34575655/89375776-02f09500-d6b4-11ea-9b96-66cb2844345d.png)

**Comment by P3NG00** [comment]
Does your console give any errors?
You're using v0.4.0?
Are you using any other mods?

♥️

**Comment by DaddySpin** [comment]
> Does your console give any errors?
> You're using v0.4.0?
> Are you using any other mods?
> 
> ♥️

Console does not give any errors that i can see.
I am using 0.4.0 and i am using about 90 other mods.

**Comment by P3NG00** [comment]
I'm sorry you're having this issue :( Are you able to give me a little more information so that I can look a little further into this?

If you're able to get these to me so I can help figure out this issue that would be fantastic!
A full server console log (uploaded to Pastebin.com, please!)
A full list of mods you're using (screenshots of folder will do as long as i can see complete file names).

(Any sensitive information you feel the need to censor like a name or IP, go ahead.)

So far I've not been able to replicate this issue. What server host are you using? Also, how did you pinpoint that it was my mod causing this?

♥ ~ P3

**Comment by DaddySpin** [comment]
I am using pebblehost for my server host and the way i narrowed down to your mod was i removed all my recently updated mods and added them back one by one until the server started using 100% cpu.

Pebblehost Console Log: https://mcpaste.io/acd5c7e2147d6049
Latest Log: https://pastebin.com/LLU6kf45

Mod List:
![image](https://user-images.githubusercontent.com/34575655/89488632-24ae5280-d76e-11ea-8964-5ea2189e2605.png)
![image](https://user-images.githubusercontent.com/34575655/89488671-3c85d680-d76e-11ea-9d18-32d69c1b2334.png)
![image](https://user-images.githubusercontent.com/34575655/89488690-460f3e80-d76e-11ea-8731-964a97bf845f.png)
![image](https://user-images.githubusercontent.com/34575655/89488711-53c4c400-d76e-11ea-85aa-c34c9a656bac.png)

**Comment by P3NG00** [comment]
Unfortunately I am unable to replicate this issue

**Comment by DaddySpin** [comment]
Well thats unfortunate

**Comment by DaddySpin** [comment]
From what i have seen it runs fine on singleplayer, but once i use it on my server it starting using excess cpu.

**Comment by P3NG00** [comment]
What I have done to test this is by creating a fabric server on my local pc. I monitor my cpu thru the task manager and it doesnt go above 10%. I've tried this eith fabric no mods, fabric + netherite horse armor, and fabric + netherite + enderite. Im not sure if maybe it has to do with a conflict somewhere? Or maybe it's the server host's issue? I've tried to make sure it's not my issue or my codes issue either. I'm not sure. I wish I could get to the bottom of this.
